### ISSUE TITLE
Mark as finish button enabled while bot run in progress

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#875]
To reproduce:

1. For a draft test plan that does not have NVDA reports, start a bot run from the report status dialog.
2. While in the initial start and jobs are queued, open the manage bot run dialog.

Actual result: The mark as finished button is visible and enabled. Activating it opens a confirmation dialog. Confirming closes the dialog and returns to the test queue. Fortunately, nothing appears changed in the test queue.

![screenshot of test queue showing initial status](https://github.com/w3c/aria-at-app/assets/7241781/aea1006f-c7d9-4f33-b596-d051e94df78e)

![screenshot of manage bot run dialog with mark as finish button enabled](https://github.com/w3c/aria-at-app/assets/7241781/604887c6-6c19-48b3-8fe6-a9640915fd18)

Expected result: Button is present but disabled.

From #732:

>Enabled when: all collection jobs are "complete" and all assertions have a verdict assigned
